src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -28,7 +28,7 @@
   @Test
   public void adsAllSlotsOnPage() {
     Page page = new Page("project43", "SyntheticTests/Mercury/Slots/AllSlots");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)